### PR TITLE
JW-546 enabled production build

### PR DESCRIPTION
**Pull request by michal-clapinski**

### CODE REVIEW

**Comment by michal-clapinski** [comment]
Retest this please

File: test/atomic-test/atomic-test.vcxproj [hunk 1]
```diff
@@ -38,7 +38,7 @@
     <ProjectGuid>{621A6695-3A8D-4F94-942B-D8745888E768}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>atomic-test</RootNamespace>
-    <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
```
Comment (PawelMarchewkaCodiLime) [inline test/atomic-test/atomic-test.vcxproj]: Why this has changed ?

**Comment by PawelMarchewkaCodiLime** [comment]
Retest this please

**Comment by michal-clapinski** [comment]
Retest this please

File: vRouter.vcxproj [hunk 2]
```diff
@@ -485,4 +523,4 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets" />
-</Project>
+</Project>
```
[the remark this answers is not in the document]
Comment (michal-clapinski) [reply vRouter.vcxproj]: Visual Studio automatically deletes it. We never review this change.

**Pull request merged**